<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>LogAccount</class>
 <widget class="QWidget" name="LogAccount" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>@product@ - Login</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="loginLabel" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>10</red>
         <green>36</green>
         <blue>106</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>233</red>
         <green>231</green>
         <blue>227</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>10</red>
         <green>36</green>
         <blue>106</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string>&lt;span style="font-size:16pt;">Login&lt;/span>&lt;br>&lt;br>&lt;span>Please choose a previously used profile&lt;br/> or fill in the informations to connect to a&lt;br/> @company@ account&lt;/span></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent" >
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>86</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>85</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="createNewAccount" >
       <property name="text" >
        <string>Configure a &amp;New Profile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>41</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/help.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LinkLabel" name="helpLabel" >
       <property name="text" >
        <string>Need help?</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton" >
       <property name="text" >
        <string>&amp;Connect</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QComboBox" name="accountComboBox" >
       <property name="editable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="passwordName" >
       <property name="text" >
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="modifyButton" >
       <property name="text" >
        <string>&amp;Modify</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="accountName" >
       <property name="text" >
        <string>Account:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="passwordLineEdit" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="LinkLabel" name="forgotPasswordLabel" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Forgot your @company@ password?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>LinkLabel</class>
   <extends>QLabel</extends>
   <header>qtutil/LinkLabel.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>createNewAccount</tabstop>
  <tabstop>accountComboBox</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>modifyButton</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
